### DISCUSSION TITLE
Release container images regularly even without new Vaultwarden version

### DISCUSSION THREAD

**Discussion by ondrejmo** [dani-garcia/vaultwarden#4591] [Ideas]
I recently started to monitor the vulnerabilities in my cluster and I was kind of surprised that one the most CVE ridden images was Vaultwarden. I believe that this has nothing with Vaultwarden itself, rather they are contained in the dependencies and the image base.

It would be nice if it were possible to build and release new container images in regular intervals (e.g. monthly) even when there is no new version, so that the vulnerabilities in the included packages are patched faster.

**Comment by BlackDex** [comment]
I was working on something like that. But haven't gotten around to it yet. I wanted to use the same release workflow but bumped into some issues.

You can at least try to use the `testing` tagged images, or the Alpine images which have lesser CVE's to begin with
